utils/Utils.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+
+/**
+ * This class includes the utitity functions.
+ * @class
+ * @classdesc Provides the common utility functions.
+ */
+class Utils {
+  /**
+   * Helper function to send ethereum transaction.
+   *
+   * @param {Object} tx Transaction object.
+   * @param {Object} tx Transaction options.
+   *
+   * @returns {Promise} Promise object.
+   */
+  static async sendTransaction(tx, txOption) {
+    return new Promise(async (onResolve, onReject) => {
```

Comment:
Is it really required to construct a new promise instead of using the one from `tx.send()`?

Reply:
I am not sure, I have taken this from mosaic.js

Reply:
It seems redundant to me 🤔